app/src/main/res/values-it/strings.xml

```diff
@@ -66,4 +66,28 @@ Occhio a non bloccare la fuoriuscita dell'aria dal coffee brewer"</string>
     <string name="tereszkiewicz_credits_subtitle">Guarda il suo Dribbble</string>
     <string name="settings_step_sound_item">Emetti un suono tra i passaggi</string>
     <string name="recipe_create_title">Aggiungi ricetta</string>
+
+
```

Comment:
Could you remove empty new lines from here? You can do that even from here \/
```suggestion
```

Reply:
removed the new lines!